Gem/Code/Source/Components/NetworkMatchComponent.h

```diff
@@ -7,28 +7,32 @@
 
 #pragma once
 
+#include <PlayerIdentityBus.h>
 #include <AzCore/EBus/ScheduledEvent.h>
 #include <Source/AutoGen/NetworkMatchComponent.AutoComponent.h>
 
 namespace MultiplayerSample
 {
     class NetworkMatchComponent
         : public NetworkMatchComponentBase
+        , public PlayerIdentityNotificationBus::Handler
     {
     public:
         AZ_MULTIPLAYER_COMPONENT(MultiplayerSample::NetworkMatchComponent, s_networkMatchComponentConcreteUuid, MultiplayerSample::NetworkMatchComponentBase);
 
         static void Reflect(AZ::ReflectContext* context);
+        
+        void OnActivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
+        void OnDeactivate(Multiplayer::EntityIsMigrating entityIsMigrating) override;
 
-        void OnInit() override {};
```

Comment:
this method seems to have disappeared completely (not in corresponding .cpp file). Is it not required to override? 
I notice we're keeping it, but not overriding any of its functionality, in `PlayerIdentityComponent`. Is there a reason this isn't the same? If not, feels like we should do one or the other in both

Reply:
It is optional to override OnInit(). Either way works, often times I remove it to keep the code shorter.